src/features/events/store.ts

```diff
@@ -578,6 +483,58 @@ const eventsSlice = createSlice({
   },
 });
 
+function addEventToState(state: EventsStoreSlice, events: ZetkinEvent[]) {
```

Comment:
Very handy to have this function here, so we know for sure that all the right things are done every time an event needst to be added to the state :)

Reply:
Thank you for pointing out the redundant code!